### PR TITLE
Update to newer version of parse5

### PR DESCRIPTION
**Pull request by RReverser**
- {setQuirksMode, isQuirksMode} => {setDocumentMode, getDocumentMode}
 - Use node type enums directly from document implementation

### CODE REVIEW

**Comment by nrkn** [comment]
Awesome. I have some comments before I merge, will add shortly

File: index.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 'use strict'
 
-const nodeType = require( 'nodetype-enum' )
```
Comment (nrkn) [inline index.js:3]: Need to uninstall from package.json as well

**Review by nrkn — CHANGES_REQUESTED** [review]

**Comment by nrkn** [comment]
I'd actually forgotten about this project. I'll add tests post merge.

File: index.js (hunk 1 above)
Comment (nrkn) [reply index.js:3]: I'll do this

**Review by nrkn — APPROVED** [review]

**Comment by nrkn** [comment]
I'm adding tests - I had to revert insertAdjacentText as it's not supported in jsdom yet

**Comment by RReverser** [comment]
@nrkn Yeah but in real DOM difference is significant, so you might want to feature detecr (also even without insertAdjacentText there is more performant way to do it).

**Comment by nrkn** [comment]
@RReverser do you want to do a PR? If not I'll add feature detect but can't be bothered optimizing

**Comment by RReverser** [comment]
@nrkn Yeah I'll do one more...

**Pull request merged**